### ISSUE TITLE
Icons won't show in bookmark toolbar [Suggestion / Bug Fix]

### ISSUE THREAD

**Issue by Sei-lus** [xiaoxiaoflood/firefox-scripts#77]
Suggestion / Bug Fix

When I put userChromeJS Manager and/or Extension Options Menu in the bookmark toolbar instead of the regular UICustomization toolbar their icon shows as a dot:

![Screenshot 2021-04-14 152223](https://user-images.githubusercontent.com/57790936/114767100-c43ccf80-9d56-11eb-987d-4af0d7814f20.png)

**Comment by xiaoxiaoflood** [comment]
I couldn't reproduce the issue. Moved EOM and ucJS Manager buttons to Bookmarks Toolbar and their icons are displayed normally, without a dot. Developer Edition 88.9b9.

**Comment by Sei-lus** [comment]
What do you recommend me to do to either fix it, or to replicate it in your machine?

**Comment by xiaoxiaoflood** [comment]
I can't do nothing unless you tell me steps to reproduce the issue. As I said, moving button to Bookmarks Toolbar is flawless here.

**Comment by Sei-lus** [comment]
@xiaoxiaoflood to be honest, I have no idea "how" to reproduce since I haven't done anything special. Installed the JS loader as per your instructions and the scripts themselves. I didn't install your whole extensive list of improvements, just a few because I didn't want to have too many things I don't use.

But I read your reply today, since you made an effort... I'll try to see what can I do to pinpoint the issue. In the next few days.

For now, what is special about my firefox browser?
- I use privacytools.io recommendations for hardening the browser (most, but not all).
- I use Arkan user.js config recommendations (handpicked).
- I use a lot of browser extension, but they can't be the cause.
- I use Firefox usercontent.css from Quantum Nox [Handpicked because it is not completely for me as I need prefer "black" not grey dark theme (Source: Github)]
- I use Firefox userchrome.css from FlyingFox user theme (Source: Github) that I futher improved upon to my own taste.
Theorically, none of those things should be the cause... But since you don't have the same issue they MUST be the cause.
After all that was installed and tweaked, I installed your toolset.
I'm of Firefox Stable 88.

One things that I do notice is this:
![image](https://user-images.githubusercontent.com/57790936/116289791-51821a00-a782-11eb-9b15-d82cee16ba7a.png)
"Normal" addons switch to a different setup on the bookmark toolbar. Badge and stuff... It is a dynamic action, that apparently is not set on the icons for the js buttons. Sadly I don't have the ability to resolve this. Can you check if this might be something that you can relate to in your own browser? If you give me a no, I will make a new profile from scratch and see if what is "conflicting" with the icons.

Either way, I don't care because it is just cosmetic. Love your repository and I'm thankful I came across it.
Sincerely,
A.

**Comment by xiaoxiaoflood** [comment]
I just tested in a fresh new profile without any user setting. The only thing I did was install userChromeJS with `rebuild_userChrome.uc.js` and `extensionOptionsMenu.uc.js`. Both buttons display just fine when moved to Bookmarks Toolbar, both in light and dark theme.

You said you're using things to mod Fx interface. Given that it works for fine in my profile and also in a non-modded profile, it seems that the issue is not in my scripts but in your tweaks.

![image](https://user-images.githubusercontent.com/25161346/117094123-53774a80-ad39-11eb-9bbc-2ffdc60698b3.png)
